app/models/soda.rb

```diff
@@ -0,0 +1,18 @@
+class Soda < ApplicationRecord
+  belongs_to :creator, class_name: 'User'
+  belongs_to :last_editor, class_name: 'User'
+  belongs_to :country
+  has_many :photos
+  has_and_belongs_to_many :flavors
```

Comment:
We should move this discussion to another place: however we decide to implement flavors, this relation will not change. Move to the flavors branch?